src/main/app/src/Components/Landscape/Overview/OverviewLayout.tsx

```diff
@@ -40,86 +40,93 @@ const useStyles = makeStyles((theme: Theme) =>
 interface Props {
   landscapes: ILandscape[] | null | undefined;
   setSidebarContent: Function;
+  landscapesCount: Number | null | undefined;
 }
 
 /**
  * Displays all available landscapes and provides all needed navigation
  */
 
-const OverviewLayout: React.FC<Props> = ({ landscapes, setSidebarContent }) => {
+const OverviewLayout: React.FC<Props> = ({ landscapes, setSidebarContent, landscapesCount }) => {
   const classes = useStyles();
   const componentClasses = componentStyles();
+  const history = useHistory();
```

Comment:
Is it possible to use history in the parent component? So we would not have to pass the count to Overviewlayout.